packages/salesforcedx-vscode-apex-replay-debugger/src/index.ts

```diff
@@ -181,6 +182,18 @@ function registerDebugHandlers(): vscode.Disposable {
 export async function activate(context: vscode.ExtensionContext) {
   console.log('Apex Replay Debugger Extension Activated');
 
+  const sfdxCoreExt = vscode.extensions.getExtension(
```

Comment:
This imports telemetry service from core extension which is mainly used to show the telemetry message and check for user's telemetry setting. We then initialize the extension's telemetry service.